.eslintrc.js

```diff
@@ -2,30 +2,50 @@
  * @type {import('eslint').Linter.Config}
  */
 // eslint-disable-next-line no-undef
+/* eslint strict: 'off' */
 module.exports = {
```

Comment:
**issue:** Type hinting on the exported object no longer works.

**suggestion:** Remove unused `// eslint-disable-next-line no-undef` line.

**suggestion:** Use strict mode for the file, rather than turning off the lint error.

```javasript
'use strict';

/**
 * @type {import('eslint').Linter.Config}
 */
module.exports = {
```